packages/ra-ui-materialui/src/input/ArrayInput/SimpleFormIterator.tsx

```diff
@@ -49,7 +58,9 @@ export const SimpleFormIterator = (props: SimpleFormIteratorProps) => {
         fullWidth,
         sx,
     } = props;
-    const { append, fields, move, remove } = useArrayInput(props);
+    const [isOpen, setIsOpen] = useState<boolean>(false);
```

Comment:
```suggestion
    const [confirmIsOpen, setConfirmIsOpen] = useState<boolean>(false);
```

I'd rename these for clarity, because the developer doesn't know at this stage what is open.